evennia/web/website/templates/website/object_detail.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
```

Comment:
This appears to belong to a different PR?

Reply:
Eh yeah I screwed up the branching. This is an extension of #1711.